### ISSUE TITLE
FTL Failed to migrate legacy database error="pq: bigint out of range"

### ISSUE THREAD

**Issue by Redskyer** [mautrix/whatsapp#727]
Since Update 0.11.0, the bridge doesn't start anymore. It seems that the database migration has failed.

I've repeated the update and migration process 2 times with the same result. For now I'm running on 0.10.9. Any ideas what I could do?

```
Container started
Oct  5, 2024 06:17:28 INF Initializing bridge built_at=2024-10-01T21:13:09Z go_version=go1.23.1 name=mautrix-whatsapp version=0.11.0+dev.e01b1777
Oct  5, 2024 06:17:28 INF Detected legacy database, migrating... action="migrate legacy db"
Oct  5, 2024 06:17:28 FTL Failed to migrate legacy database error="pq: bigint out of range" db_section=main pq_error={"code":"numeric_value_out_of_range","file":"int8.c","line":"925","routine":"int84mul","severity":"ERROR"}
Container stopped
```

**Comment by tulir** [comment]
That's an outdated version

**Comment by Redskyer** [comment]
What do you mean with outdated version?

I've got the problems with "bridge built_at=2024-10-01T21:13:09Z go_version=go1.23.1 name=mautrix-whatsapp version=0.11.0+dev.e01b1777", which is the newest container image, I could download.

**Comment by tulir** [comment]
The newest image (available on the `:latest` tag) is 1f454a82, not e01b1777

**Comment by Redskyer** [comment]
Thanks for help. A new try today was successful. Now the correct version was downloaded by latest tag.
